pages/index.tsx

```diff
@@ -1,7 +1,114 @@
-import React, { Component } from 'react'
+import Contest from '../components/Contest'
+import Layout from '../components/Layout'
+import { getContestList } from '../services/reventService'
+import React, { useState, useEffect } from 'react'
+import SearchInput from '../components/SearchInput'
 
-export default class index extends Component {
-  render() {
-    return <div>Home Page</div>
+export default function Home(props) {
+  const { contestList } = props
+
+  const [statusFilter, setStatusFilter] = useState('All')
+
+  const [open, setOpen] = useState(false)
+
+  const filteredContestList = contestList.filter(
```

Comment:
if contentList is null this throws an error

Reply:
this should always return an array, either an empty array or an array with elements. This way you don't have to have `?` like `filteredContestList?.map((contest)`, you can just do `filteredContestList.map((contest)`